invenio_records_rest/config.py

```diff
@@ -363,3 +363,6 @@ def deleted_pid_error_handler(error):
     ),
 }
 """Handlers for ElasticSearch error codes."""
+
+RECORDS_REST_DEFAULT_SEARCH_RESULTS_SIZE = 10
+"""Default search results size.""""
```

Comment:
Should we, perhaps, move it to `RECORDS_REST_ENDPOINTS` like for example `max_result_window`? (I am really just asking)

Reply:
👍 don't have strong opinion, we'll add yet another rule in ``create_url_rules`` (it's quite big). The main idea behind this PR is to have consistency in all of your searches, without add the logic everywhere in the UI. What I mean is, having separate search results size per type it's not the real case, but we can do it. ☺️ Do you think will be useful per type?